src/client.js

```diff
@@ -1777,6 +1777,34 @@ class Client {
         const result = this._toRepresentation(doc);
         return result;
     }
+
+    /**
+     * Creates and returns a new instance of the specified entity
+     * @param entityType
+     * @param entity
+     * @returns {Promise<XML.XtkObject>}
+     */
+    async newInstance(entityType, entity) {
+        try{
+            const that = this;
+            const root = entityType.split(":").pop();
+            const xml= this._fromRepresentation(root, entity, 'SimpleJson')
```

Comment:
A space character is missing before the "=" sign. There are a couple of other linting errors. Please run linting, tests, jsdoc, and _check the results_ before submitting PRs as mentioned in the readme.